context/app/static/js/components/Search/DevSearch.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { ExistsQuery, BoolMustNot } from 'searchkit';
+import SearchWrapper from './SearchWrapper';
+import { readCookie } from '../../helpers/functions';
+import './Search.scss';
+// eslint-disable-next-line import/named
+import { field, filter, checkboxFilter } from './utils';
+import { lastModifiedSort, sizeSort } from './config';
+
+const nexus_token = readCookie('nexus_token');
+const httpHeaders = nexus_token
+  ? {
+      Authorization: `Bearer ${nexus_token}`,
+    }
+  : {};
+const searchProps = {
+  // The default behavior is to add a "_search" path.
+  // We don't want that.
+  searchUrlPath: '',
+  // Pass Globus token:
+  httpHeaders,
+  // Prefix for details links: (Entities which are not datasets will redirect.)
+  detailsUrlPrefix: '/browse/dataset/',
+  // Search results field which will be appended to detailsUrlPrefix:
+  idField: 'uuid',
+  // Search results fields to display in table:
+  resultFields: [
+    field('entity_type', 'type'),
+    field('display_doi', 'ID'),
+    field('mapped_last_modified_timestamp', 'Last Modified'),
+    field('mapper_metadata.size', 'Doc Size'),
+  ],
+  // Default hitsPerPage is 10:
+  hitsPerPage: 20,
+  // Sidebar facet configuration:
+  filters: [
+    filter('entity_type', 'Entity Type'),
+    filter('mapper_metadata.version', 'Mapper Version'),
+    checkboxFilter('has_metadata', 'Has metadata?', ExistsQuery('metadata')),
+    checkboxFilter('no_metadata', 'No metadata?', BoolMustNot(ExistsQuery('metadata'))),
+    checkboxFilter('has_files', 'Has files?', ExistsQuery('files')),
+    checkboxFilter('no_files', 'No files?', BoolMustNot(ExistsQuery('files'))),
+  ],
+  sortOptions: lastModifiedSort.concat(sizeSort),
+  queryFields: ['everything'],
+  isLoggedIn: Boolean(nexus_token),
+};
+
+function DevSearch(props) {
+  const { elasticsearchEndpoint } = props;
+  const allProps = Object.assign(searchProps, { apiUrl: elasticsearchEndpoint });
```

Comment:
```suggestion
  const allProps = {...searchProps, apiUrl: elasticsearchEndpoint};
```
We can use spread here?